src/test/scala/is/hail/testUtils/RichTable.scala

```diff
@@ -88,44 +88,11 @@ class RichTable(ht: Table) {
     ht.copy(rdd = ht.rdd.map(annotF), signature = finalSignature, key = newKey)
   }
 
-  def annotate(code: String): Table = {
-    val ec = ht.rowEvalContext()
-
-    val (paths, asts) = Parser.parseAnnotationExprsToAST(code, ec).unzip
-    if (paths.length == 0)
-      return ht
-
-    val irs = asts.flatMap { _.toIR() }
-
-    if (irs.length != asts.length || ht.globalSignature.size != 0) {
-      val (paths, types, f) = Parser.parseAnnotationExprs(code, ec, None)
-
-      val inserterBuilder = new ArrayBuilder[Inserter]()
-
-      val finalSignature = (paths, types).zipped.foldLeft(ht.signature) { case (vs, (ids, sig)) =>
-        val (s: TStruct, i) = vs.insert(sig, ids)
-        inserterBuilder += i
-        s
-      }
+  def keyByExpr(exprs: (String, String)*): Table =
+    ht.select(s"annotate(row, {${ exprs.map { case (n, e) => s"`$n`: $e" }.mkString(",") }})", Some(exprs.map(_._1).toIndexedSeq))
```

Comment:
can't just backtick it like this -- the field name could have a backtick in it. Use prettyIdentifier (I think).